server/src/com/thoughtworks/go/server/service/BackupService.java

```diff
@@ -116,14 +117,26 @@ public ServerBackup startBackup(Username username, HttpLocalizedOperationResult
             try {
                 backupRunningSince = now;
                 backupStartedBy = username.getUsername().toString();
+                LOGGER.info("Backing up GoCD version file.");
+                long start = System.currentTimeMillis();
                 backupVersion(destDir);
+                LOGGER.info(String.format("Finished backing up GoCD version file. Took {%s} ms.", System.currentTimeMillis() - start));
+                LOGGER.info(String.format("Backing up config directory."));
+                start = System.currentTimeMillis();
                 backupConfig(destDir);
+                LOGGER.info(String.format("Finished backing up config directory. Took {%s} ms.", System.currentTimeMillis() - start));
+                LOGGER.info(String.format("Backing up config.git repository"));
+                start = System.currentTimeMillis();
                 configRepository.doLocked(new VoidThrowingFn<IOException>() {
                     @Override public void run() throws IOException {
                         backupConfigRepository(destDir);
                     }
                 });
+                LOGGER.info(String.format("Finished backing up config directory. Took {%s} ms.", System.currentTimeMillis() - start));
```

Comment:
Spring has this for measuring time http://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/StopWatch.html use that if spring classes are available in the classpath